gui/gui.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
```

Comment:
This is from scratch to help developers easily test things from their own CLI to roughly mock what are docker container is going to be doing in CI

Reply:
Correction - parts of this is from the deleted build.sh file above

Reply:
I can see how this would be useful for developing testing locally without Docker. However, it does open us up to compatibility issues (e.g. when dealing with differences between Debian's Qt5 and Homebrew's.) I'm not claiming this change obviates the need for Docker, but it does make it less important. (Another way to put it: why not produce Debian packages for the GUI instead of Docker images?)